website/docs/components/reveal/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Reveal
+description: A toggle that reveals additional information or details about an element or content to the user when triggered.
+caption: A toggle that reveals additional content to the user when triggered.
+status: released
+links:
+  figma: {link to the "Reveal" page in the components Figma library}
+  github: {link to the "Reveal" component's folder in the GitHub repo}
```

Comment:
We probably have these links, yeah?

Reply:
Thanks, once @christian's sub documentation branch is merged, in I will double-check them and add them if hasn't.

Reply:
Links have now been added. (Christian added the Figma one and I added Github. :) )